pandas/sparse/tests/test_series.py

```diff
@@ -1048,6 +1051,44 @@ def _dense_series_compare(s, f):
     tm.assert_series_equal(result.to_dense(), dense_result)
 
 
+class TestSparseSeriesAnalytics(tm.TestCase):
+    def setUp(self):
+        arr, index = _test_data1()
+        self.bseries = SparseSeries(arr, index=index, kind='block',
```

Comment:
*could* make a ``class Base(object)`` in this file where we define ``setUp`` (then inherit this for the classes). If that is more clear.